packages/paste-website/src/pages/blog/2022-07-26-paste-newsletter.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: Paste Design System Newsletter - July 2022 Edition
+slug: /blog/2022-07-26-paste-newsletter/
+date: 2022-07-26
+author: Paste
+avatar:
+author_link:
+status: published
+external_link:
+excerpt: What's new in Paste in July 2022? New tokens list page, Figma Slot components, Chat Log documentation, and more!
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Box} from '@twilio-paste/box';
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {UnorderedList, ListItem} from '@twilio-paste/core/list';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+
+export const pageQuery = graphql`
+  {
+    mdx(frontmatter: {slug: {eq: "/blog/2022-07-26-paste-newsletter/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+        date(formatString: "MMMM D, YYYY")
+        machineDate: date
+        author
+        avatar
+        author_link
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ArticleHeader
+  title={props.data.mdx.frontmatter.title}
+  description={props.data.mdx.frontmatter.description}
+  date={props.data.mdx.frontmatter.date}
+  machineDate={props.data.mdx.frontmatter.machineDate}
+  author={props.data.mdx.frontmatter.author}
+  avatar={props.data.mdx.frontmatter.avatar}
+  authorLink={props.data.mdx.frontmatter.author_link}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} hideFeedback />
+
+<ArticleContent>
+
+## 📣 What's new in Paste
+
+### The new tokens page is live!
+
+![](../../assets/images/articles/2022-07-26-paste-newsletter/tokens-page.gif)
```

Comment:
This image is broken now, but im not sure why. The image path is correct and we should be able to use gifs like this.